plugins/grype/surfactantplugin_grype.py

```diff
@@ -4,34 +4,55 @@
 # SPDX-License-Identifier: MIT
 import gzip
 import subprocess
+
+# Configure loguru to show logs at DEBUG level or higher
+import sys
 import tempfile
 from typing import List, Optional
 
 from loguru import logger
 
 import surfactant.plugin
+from surfactant.plugin.manager import (  # Import the helper function
+    get_plugin_manager,
+    is_plugin_blocked,
+)
 from surfactant.sbomtypes import SBOM, Software
 
+logger.remove()  # Remove the default logger configuration
+logger.add(sys.stdout, level="DEBUG")  # Add a new logger that outputs to console
```

Comment:
Does this mess up the logger used in the rest of Surfactant? https://github.com/LLNL/Surfactant/blob/main/surfactant/__main__.py#L55-L5

Instead of changing the logger settings here, in the tests when you run Surfactant you can pass it the --log-level option.